src/components/footer.js

```diff
@@ -1,5 +1,58 @@
+import {
+  Center,
+  Container,
+  Divider,
+  Flex,
+  Image,
+  Link,
+  Text,
+  Box,
+} from "@chakra-ui/react";
 import React from "react";
+import { Link as ReactLink } from "react-router-dom";
+import LinkGrid from "./footer-link-grid";
 
 export function Footer() {
-  return <h1> This is Footer</h1>;
+  return (
+    <>
+      <Center bg="#E2E8F0" w="100%" position="relative" bottom="0" minH="320px">
+        <Container maxW="container.xl">
+          <Flex mt="3em" justify="space-between" align="center">
+            <Link variant="footerLink" as={ReactLink} to="/">
+              <Flex align="center">
+                <Image
+                  boxSize="100px"
+                  src="https://i.postimg.cc/rm8LXXgw/Asset-1.png"
+                  alt="Website logo"
+                ></Image>
+                <Text fontWeight="bold" fontSize="3xl" pl={4} pb={2}>
+                  movieFreak
+                </Text>
+              </Flex>
+            </Link>
+            <LinkGrid />
+          </Flex>
+          <Divider m="2em 0" orientation="horizontal" borderColor="gray.400" />
```

Comment:
Don't use margin here, instead use the `spacing` prop from the parent to control the spacing between elements. You can just wrap all the elements in a `<VStack />`, `<Flex />`, or something else.